### ISSUE TITLE
Hide Attributes

### ISSUE THREAD

**Issue by ddorato** [joe-ng/homebridge-dyson-link#31]
Is there a way to hide attributes like night mode or Jet Focus?  I don't see much need in my flows to control that via Siri or HomeKit.

**Comment by joe-ng** [comment]
Yup, i just added this as an option in the config file like below, you may check the readme file for details

      // This controls the visibility of Night Mode button, default to true
      "nightModeVisible" : true,

      // This controls the visibility of Jet Focus button, default to true
      "focusModeVisible" : true

**Comment by jausto** [comment]
I tried this and it didn't remove them.  "focusModeVisible" : false. Is because i change their names in homekit?

**Comment by ddorato** [comment]
Same.  Even after flushing the cached accessories.

**Comment by joe-ng** [comment]
Just released a fix for this. please try the latest version

**Comment by michaeljdeeb** [comment]
Just wanted to confirm that `"nightModeVisible" : false` is now working correctly for me as of 2.1.3.
